thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js

```diff
@@ -372,16 +372,16 @@ export default Component.extend({
 
       if (anomalies) {
         anomalies.forEach(a => {
-          const change = (a.avgBaselineVal !== 0) ? (a.avgCurrentVal/a.avgBaselineVal - 1.0) * 100.0 : 0;
+          const change = (a.avgBaselineVal !== 0 && a.avgBaselineVal !== "Infinity" && a.avgCurrentVal !== "Infinity") ? (a.avgCurrentVal/a.avgBaselineVal - 1.0) * 100.0 : 0;
           let tableRow = {
             anomalyId: a.id,
             metricUrn: a.metricUrn,
             start: a.startTime,
             end: a.endTime,
             startDateStr: this._formatAnomaly(a),
             durationStr: getFormattedDuration(a.startTime, a.endTime),
-            shownCurrent: humanizeFloat(a.avgCurrentVal),
```

Comment:
E.g, for rule based detection if the previous value is 0 then the change would be infinity. @harleyjj  is it possible to show it as "n/a" or "-"?

Reply:
This particular part is fed into the table on the Alert-Preview component (displayed in Preview and Alert Overview).  I could put some other kind of text, but it should probably be small since the field is usually a number.  And yes, the backend is returning it in the anomaly response.